scripts/xtask.rs

```diff
@@ -16,54 +19,13 @@ enum Args {
     /// Note that we only warn on our _direct_ dependencies,
     /// not the entire supply chain.
     Outdated,
+    /// Execute proving via RPC endpoint.
+    ProveRpc(Box<ProveRpcArgs>),
```

Comment:
info: did clippy insist that you `Box` this?

Reply:
Yea the typical complaint about lop-sided enum variants